common/changes/office-ui-fabric-react/arkgupta-dragDropBorderFix_2018-11-22-18-06.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "prevent override of border while dragging",
```

Comment:
We've been trying to improve our changelog notes by including the component name as a prefix for these comment values, here's an example: https://github.com/OfficeDev/office-ui-fabric-react/pull/7199/files#diff-6a595f15feecd9c3e14bd6c78fea676bR5

Consider the following as your comment value:

>DetailsList: Prevent override of column cell border while dragging.